packages/pwa-buildpack/lib/Utilities/graphQL.js

```diff
@@ -0,0 +1,63 @@
+const fetch = require('node-fetch');
+const graphQLQueries = require('../queries');
+
+const fetchQuery = query => {
+    return fetch(
+        new URL('graphql', process.env.MAGENTO_BACKEND_URL).toString(),
+        {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+                'Accept-Encoding': 'gzip'
+            },
+            body: JSON.stringify({ query })
+        }
+    )
+        .then(result => result.json())
+        .then(json => json.data)
+        .catch(err => {
+            console.error(err);
+            throw err;
```

Comment:
Note that instead of returning an empty string here we are now re-throwing the caught error.

This allows the caller to determine how they want to handle the error case in a more idiomatic way `try / catch` instead of having to check success cases against an empty string.